package.json

```diff
@@ -1,54 +1,29 @@
 {
   "name": "hubot",
   "version": "2.19.0",
-  "author": "hubot",
   "keywords": [
     "github",
     "hubot",
     "campfire",
     "bot"
   ],
   "description": "A simple helpful robot for your Company",
-  "license": "MIT",
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/github/hubot.git"
   },
   "dependencies": {
     "hubot": "^2.19.0",
-    "async": ">=0.1.0 <1.0.0",
-    "chalk": "^1.0.0",
-    "cline": "^0.8.2",
-    "coffee-script": "1.6.3",
-    "connect-multiparty": "^1.2.5",
-    "express": "^3.21.2",
-    "log": "1.4.0",
-    "optparse": "1.0.4",
-    "hubot-front": "^0.0",
-    "scoped-http-client": "0.11.0"
+    "hubot-duplicate": "github:resin-io-modules/hubot-duplicate#alpha",
+    "hubot-flowdock": "github:resin-io-modules/hubot-flowdock#alpha"
   },
   "devDependencies": {
     "resin-lint": "^1.4.0",
-    "hubot-mock-adapter": "^1.0.0",
-    "hubot-flowdock": "",
-    "hubot-slack": "",
-    "coffee-errors": "0.8.6",
-    "mocha": "^2.1.0",
-    "mockery": "^1.4.0",
-    "sinon-chai": "^2.8.0",
-    "sinon": "~1.17.0",
-    "chai": "~2.1.0"
```

Comment:
We've lost all these testing dependencies, but we've still got all the tests that use them. One of those two is wrong.